web/gds-user-ui/src/components/OrganizationProfile/TrisaImplementation.tsx

```diff
@@ -36,7 +36,7 @@ const TrisaImplementation: React.FC<TrisaImplementationProps> = ({ data }) => {
             <Tbody
               sx={{
                 '*': {
-                  fontSize: '1rem',
+                  fontSize: '16px',
```

Comment:
It was decided to use `px` here after noting that the font size for some of the data was displaying as `14px` while other text was at `16px`.